visdag/visds_config.yaml

```diff
@@ -0,0 +1,11 @@
+solids:
+    visds_solid:
+      db_config:
+        - DB_TYPE=postgresql
+        - POSTGRES_USER=viper
+        - POSTGRES_PASSWORD=viper
```

Comment:
This file should not be in the repo.  You could include a visdag/visds_config.example.yml in the repo to show people what is expected, but the install instructions should instruct a user to create their own visdag/visds_config.yml from the example file.

Reply:
Missed this file when changing to .env configuration. Removed file. 

A TODO is to update the install instructions. I want to get this PR done first, so will not add a new file yet.